packages/botonic-plugin-flow-builder/src/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { HtMediaFileLocale } from './hubtype-models'
+
+export function getImageByLocale(locale: string, image: HtMediaFileLocale[]) {
+  const result = image.find(t => t.locale == locale)
+  return result?.file ?? ''
+}
```

Comment:
You could move this to the `FlowImage` class as an static method, similar to the `getTextByLocale`

Reply:
changed 👌